eq-author/src/App/QuestionnairesPage/QuestionnairesTable/Row.js

```diff
@@ -93,47 +115,60 @@ class Row extends React.Component {
   render() {
     const { questionnaire, onDeleteQuestionnaire, ...rest } = this.props;
     const isOptimisticDupe = this.isQuestionnaireADuplicate();
+    const { showDeleteQuestionnaireDialog } = this.state;
+    console.log(questionnaire.displayName, onDeleteQuestionnaire, rest);
 
     return (
-      <FadeTransition
-        {...rest}
-        enter={isOptimisticDupe}
-        exit={!isOptimisticDupe}
-      >
-        <TR innerRef={this.rowRef} disabled={isOptimisticDupe}>
-          <TD>
-            <TruncatedQuestionnaireLink
-              data-test="anchor-questionnaire-title"
-              questionnaire={questionnaire}
-              title={questionnaire.title}
-              disabled={isOptimisticDupe}
-            >
-              {questionnaire.title}
-            </TruncatedQuestionnaireLink>
-          </TD>
-          <TD>
-            <FormattedDate date={questionnaire.createdAt} />
-          </TD>
-          <TD>
-            <Truncated>{questionnaire.createdBy.name || "Unknown"}</Truncated>
-          </TD>
-          <TD textAlign="center">
-            <ButtonGroup>
-              <DuplicateButton
-                data-test="btn-duplicate-questionnaire"
-                onClick={this.handleDuplicateQuestionnaire}
-                disabled={isOptimisticDupe}
-              />
-              <IconButtonDelete
-                hideText
-                data-test="btn-delete-questionnaire"
-                onClick={partial(onDeleteQuestionnaire, questionnaire.id)}
+      <>
+        <DeleteConfirmDialog
+          isOpen={showDeleteQuestionnaireDialog}
+          onClose={this.handleCloseDeleteQuestionnaireDialog}
+          onDelete={this.handleDeleteQuestionnaire}
+          title={questionnaire.title}
+          alertText="Questionnaire will be deleted."
```

Comment:
Do we want to also add that all sections and pages will be deleted to have it consistent with the section delete modal?